client/lib/query-matcher.ts

```
@@ -397,6 +875,157 @@ ${data.activeSprints
+
+        return `**Combined Search Results:**\n\`\`\`json\n${JSON.stringify(data, null, 2)}\n\`\`\``;
+
+      case "fetch_perplexity_data":
+        // Handle Perplexity search results
+        if (data.error) {
+          return `⚠️ **Search Error:** ${data.error}`;
+        }
+
+        const searchContent = data.content || "";
+        const citations = data.citations || [];
+        const sources = data.sources || [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)